modules/installation-configuration-parameters.adoc

```diff
@@ -1159,6 +1203,31 @@ The GCP Compute Engine System service account email, like `<service_account_name
 |For compute machines, the ID of the project in which the KMS key ring exists. This value defaults to the VM project ID if not set.
 |The GCP project ID.
 
+|`compute.platform.gcp.osDisk.encryptionKey.kmsKeyServiceAccount`
+|The GCP service account used for the encryption request for compute machines. If absent, the Compute Engine default service account is used. For more information on GCP service accounts, see Google's documentation on link:https://cloud.google.com/compute/docs/access/service-accounts#compute_engine_service_account[service accounts].
+|The GCP service account email, for example `<service_account_name>@<project_id>.iam.gserviceaccount.com`.
+
+|`compute.platform.gcp.osDisk.diskSizeGB`
+|The size of the disk in gigabytes (GB). This value applies to compute machines.
+|Any size between 16 GB and 65536 GB.
+
+|`compute.platform.gcp.osDisk.diskType`
+|The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type] for compute machines.
+|Either the default `pd-ssd` or the `pd-standard` disk type.
+
+|`compute.platform.gcp.tags`
+|Optional. Additional network tags to add to the compute machines. If set, this parameter overrides the `platform.gcp.defaultMachinePlatform.tags` parameter for compute machines.
+|One or more strings, for example "compute-network-tag1".
```

Comment:
```suggestion
|One or more strings, for example `"compute-network-tag1"`.
```
Same comment for similar situations.

Reply:
With a string value that doesn't need to be in quotation marks in the configuration file, should I list it as `compute-network-tag` or `"compute-network-tag"`?

Reply:
Oh, yeah, in that case, no quotes. :+1: